src/frame.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+class Frame {
+  constructor() {
+    this.scores = [];
+    this.frameScore = 0;
+  }
+
+  add(num) {
+    this.scores.push(num);
+    this.frameScore += num;
+  }
+
+  total() {
+    return this.frameScore
+  }
+
+  isStrike() {
+    return this.scores[0] === 10 && this.frameLength() < 3
+  }
+
+  isSpare() {
+    const scoreCheck = this.scores[0] + this.scores[1] === 10
+    return this.frameLength() === 2 && scoreCheck
+  }
+
```

Comment:
clear definition of 'Strike' and 'Spare'. And great way for clarifying  the 'scoreCheck' value with 'const', making it unchangeable.